api/v1alpha1/instaslice_types.go

```diff
@@ -20,6 +20,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// Mig defines the MIG slice details
```

Comment:
I think it's better to elaborate the comment here so that the same would be reflected in the go doc